### PR TITLE
Update rubyslim to work with Ruby 1.9.3

### PR DESCRIPTION
**Pull request by marosluuce**
- Removed references to jcode.
- Updated tests to use `# coding: utf-8`

Tests pass in 1.9.3 and even in 2.0.0. I'm not sure about 1.8.7 though. Does compatibility need to be maintained?

### CODE REVIEW

File: bin/rubyslim [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env ruby
 
+$:.unshift(File.expand_path(File.dirname(__FILE__) + "/../lib/"))
```
Comment (dougbradbury) [inline bin/rubyslim:3]: Why is this necessary?  I think ruby slim requires the writer of the test to put the rubyslim lib on the path via a fitnesse variable. Is this an attempt to eliminate this need?

**Comment by dougbradbury** [comment]
The core team has dropped support for Ruby 1.8.7 so I think it fair if we do as well.  Perhaps we should create a branch and let it go on living that supports 1.8.7 but add no new features to it.

**Comment by marosluuce** [comment]
The branch idea sounds good. As for line 3, that was so that I didn't have to add a path entry for rubyslim. I had already set TEST_RUNNER to bin/rubyslim and felt it was redundant to neet to also specify the path to lib. I'm open to suggestions if there's a better way to do this.

**Comment by onipirate** [comment]
hi all,

I am geeting this error when running rubyslim in fitnesse. I'm a bit stuck. can anyone help?

!define COLLAPSE_SETUP {true}
!define COLLAPSE_TEARDOWN {true}
!define TEST_SYSTEM {slim}
!define TEST_RUNNER {C:\Ruby193\lib\ruby\1.9.1\rubyslim\ruby_slim.rb}
!define COMMAND_PATTERN {ruby -w -v -I %p %m}
!define PATH_SEPARATOR { -I }
!path C:\

C:/Ruby193/lib/ruby/1.9.1/rubyslim/statement_executor.rb:66: warning: assigned but unused variable - e
C:/Ruby193/lib/ruby/1.9.1/rubyslim/statement_executor.rb:73: warning: `&' interpreted as argument prefix
C:/Ruby193/lib/ruby/1.9.1/rubyslim/statement_executor.rb:160: warning: shadowing outer local variable - match
C:/Ruby193/lib/ruby/site_ruby/1.9.1/rubygems/custom_require.rb:36: warning: loading in progress, circular require considered harmful - C:/Ruby193/lib/ruby/1.9.1/rubyslim/statement.rb
    from C:/Ruby193/lib/ruby/1.9.1/rubyslim/ruby_slim.rb:4:in`<main>'
    from C:/Ruby193/lib/ruby/site_ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
    from C:/Ruby193/lib/ruby/site_ruby/1.9.1/rubygems/custom_require.rb:36:in`require'
    from C:/Ruby193/lib/ruby/1.9.1/rubyslim/list_executor.rb:1:in `<top (required)>'
    from C:/Ruby193/lib/ruby/site_ruby/1.9.1/rubygems/custom_require.rb:36:in`require'
    from C:/Ruby193/lib/ruby/site_ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
    from C:/Ruby193/lib/ruby/1.9.1/rubyslim/statement.rb:1:in`<top (required)>'
    from C:/Ruby193/lib/ruby/site_ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
    from C:/Ruby193/lib/ruby/site_ruby/1.9.1/rubygems/custom_require.rb:36:in`require'
    from C:/Ruby193/lib/ruby/1.9.1/rubyslim/statement_executor.rb:2:in `<top (required)>'
    from C:/Ruby193/lib/ruby/site_ruby/1.9.1/rubygems/custom_require.rb:36:in`require'
    from C:/Ruby193/lib/ruby/site_ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
